Dockerfile

```diff
@@ -3,14 +3,15 @@ FROM jenkins:1.642.4
 USER root
 
 RUN echo "deb http://archive.debian.org/debian jessie main" > /etc/apt/sources.list \
-    && rm /etc/apt/sources.list.d/jessie-backports.list
+    && rm /etc/apt/sources.list.d/jessie-backports.list \
+    && sed -i 's;http://archive.debian.org/debian/;http://deb.debian.org/debian/;' /etc/apt/sources.list
 
 RUN apt-get update \
-    && apt-get install -y apt-transport-https ca-certificates \
-    && apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
-    && echo deb https://apt.dockerproject.org/repo debian-jessie main >> /etc/apt/sources.list.d/docker.list \
+    && apt-get install --force-yes -y apt-transport-https ca-certificates curl gnupg-agent software-properties-common sudo \
```

Comment:
`--force-yes` and `-y` are redundant, no?

Reply:
not really, it does not pass through all the errors

Reply:
ok but then do you need it? It's generally not recommended...